Slider/Slider.js

```diff
@@ -181,6 +181,14 @@ const SliderBase = kind({
 		 */
 		progressAnchor: PropTypes.number,
 
+		/**
+		 * Displays an anchor at `progressAnchor`.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		showAnchor: PropTypes.bool,
```

Comment:
The Slider in some skins doesn't seem to support this prop. Could you check it?
![image](https://user-images.githubusercontent.com/4239873/102961772-8b1eec00-4528-11eb-9747-fc005664daa4.png)

Reply:
thanks for the question. Indeed, from the sampler, it looks like the skin does not support this feature.
Cobalt, Copper and Titanium sliders have the fill-color set to `transparent` and the `anchor` has the same color as the slider fill color.
Do you think it would be better to show the anchor anyway? Here's my example: 

![image](https://user-images.githubusercontent.com/63335068/102973397-52b9e680-4505-11eb-86d0-2f7e8d3c3656.png)